### ISSUE TITLE
Deprecate Makefile

### ISSUE THREAD

**Issue by mrueg** [drone/drone#1813]
Currently the Makefile assumes that GOPATH is set to a single directory, instead there could be multiple directories it could be set to concatenated with : 

https://github.com/drone/drone/blob/master/Makefile#L49

Can the GOPATH be replaced with anything that works also for this case? Thanks!

**Comment by bradrydzewski** [comment]
I'm re-naming the issue. I think the ideal solution would be to make drone go gettable so that we don't need a makefile or scripts that rely on the GOPATH for the build process.

**Comment by tboerger** [comment]
I would suggest fileb0x instead of bindata :)

**Comment by appleboy** [comment]
@tboerger Good suggestion https://github.com/UnnoTed/fileb0x

**Comment by metalmatze** [comment]
I'm using https://github.com/gobuffalo/packr lately. Just to put it on the list too.

**Comment by bradrydzewski** [comment]
Thanks, I will definitely check out packr and filebox for drone-ui 😄 

For drone/drone we actually don't store any binary assets. They are all text assets related to database queries, migration and templates. So I'm using https://github.com/bradrydzewski/togo

This is a tool of my own creation that handles embedding plain text files as plain text (strings) and generating the relevant template, migration, sql code. This is an example of what generated template code looks like, for those interested:

```Go
package template

import "html/template"

// list of embedded template files.
var files = []struct {
	name string
	data string
}{
	{
		name: "en.tmpl",
		data: en,
	}, {
		name: "es.tmpl",
		data: es,
	},
}

// T exposes the embedded templates.
var T *template.Template

func init() {
	T = template.New("_").Funcs(funcMap)
	for _, file := range files {
		T = template.Must(
			T.New(file.name).Parse(file.data),
		)
	}
}

//
// embedded template files.
//

// files/en.tmpl

var en = `<html>
<body>
Hello world
</body>
</html>
`

// files/es.tmpl

var es = `<html>
<body>
Hola Mundo
</body>
</html>
`
```

**Comment by bradrydzewski** [comment]
merged #2040
